scripts/config.py

```diff
@@ -485,8 +485,9 @@ def write(self, filename=None):
     def main():
         """Command line mbedtls_config.h manipulation tool."""
         parser = argparse.ArgumentParser(description="""
-        Mbed TLS configuration file manipulation tool.
-        """)
+                                        Mbed TLS configuration file manipulation tool.
```

Comment:
Pylint is unhappy about the indentation.

To reproduce the CI failure:

```
scripts/min_requirements.py
tests/scripts/check-python-files.sh
```

You'll probably want to do this in a virtual environment with a Python version that isn't too recent, otherwise Pylint complains more. Just make sure

```
pylint scripts/config.py
```

doesn't complain about indentation anymore.